server.go

```diff
@@ -1848,10 +1848,36 @@ func (s *Server) planSelectStatement(stmt *influxql.SelectStatement, database st
 	s.mu.RLock()
 	defer s.mu.RUnlock()
 
-	// Find database.
-	db := s.databases[database]
-	if db == nil {
-		return nil, ErrDatabaseNotFound
+	// If this is a wildcard statement, expand this to use the fields
+	// This is temporary until we move other parts of the system further
+	isWildcard := false
+	for _, f := range stmt.Fields {
+		if _, ok := f.Expr.(*influxql.Wildcard); ok {
+			isWildcard = true
+			break
+		}
+	}
+
+	if len(stmt.Fields) != 1 && isWildcard {
+		return nil, fmt.Errorf("unsupported query: %s.  currently only single wildcard is supported.", stmt.String())
```

Comment:
Nit: This sounds like `SELECT *,value` is supported, since it's still a single wildcard. :-) "....only solitary wildcard supported."?